server/libs/message-bus/src/test/scala/cool/graph/messagebus/pubsub/inmemory/BasicPerformanceTesting.scala

```diff
@@ -0,0 +1,104 @@
+package cool.graph.messagebus.pubsub.inmemory
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import akka.actor.{Actor, ActorSystem, Props}
+import akka.stream.ActorMaterializer
+import cool.graph.messagebus.pubsub.PubSubProtocol.Publish
+import cool.graph.messagebus.pubsub.{IntermediateCallbackActor, Message, PubSubRouterAlt, PubSubRouter}
+import org.scalatest.WordSpecLike
+
+import scala.reflect.ClassTag
+
+class BasicPerformanceTesting extends WordSpecLike {
+
+//  "Akka Routing implementation" should {
```

Comment:
Do we need this?

Reply:
Yes I'd keep it for future reference for now.